.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit and fetch tag history
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install release dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+
+      - name: Build and publish package
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_AZAVEA_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_AZAVEA_PASSWORD }}
```

Comment:
I've made these secrets available to this repository. I purposefully waited until after I was done testing the Release workflow to avoid accidentally publishing anything. 😬